sycl/test-e2e/DeviceLib/built-ins/offload-prec-div-sqrt.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %{build} -foffload-fp32-prec-div -foffload-fp32-prec-sqrt -o %t.out
```

Comment:
Do we need to be explicit about `-fp-model=source` for the host options?

Reply:
TBH, I don't understand the proposal. Is the idea to pass -ffp-model=precise for host (if so it feels redundant) or something different?

Reply:
Agreed to ignore for now, can follow up downstream if necessary.